editor/src/clj/editor/boot_open_project.clj

```diff
@@ -206,8 +191,9 @@
           search-results-view  (search-results-view/make-search-results-view! *view-graph*
                                                                               (.lookup root "#search-results-container")
                                                                               open-resource)
-          changes-view         (changes-view/make-changes-view *view-graph* workspace prefs async-reload!
-                                                               (.lookup root "#changes-container"))
+          changes-view         (changes-view/make-changes-view *view-graph* workspace prefs (.lookup root "#changes-container")
+                                                               (fn [changes-view moved-files]
+                                                                 (app-view/async-reload! app-view changes-view workspace moved-files)))
```

Comment:
Changed `async-reload!` callback signature to only provide the `changes-view` and the `moved-files`. This is called when files are reverted from the Changes View.

Changed argument order to `make-changes-view`, putting the `async-reload!` callback in the last argument position.